lib/posix-process/exportsyms.uk

```diff
@@ -113,3 +113,4 @@ uk_syscall_e_tgkill
 tkill
 uk_syscall_r_tkill
 uk_syscall_e_tkill
+uk_signal_deliver
```

Comment:
I guess this will be no longer needed once sysrettab is in.

Reply:
Leaving open to make sure it's not missed 👍🏼